sdk/keyvault/keyvault-certificates/src/index.ts

```diff
@@ -499,20 +556,22 @@ export class CertificateClient {
    *   name: "b",
    *   phone: "222222222222"
    * }]);
-   * await client.deleteCertificateContacts();
+   * await client.deleteContacts();
```

Comment:
Now that we are renaming `deleteCertificateContacts()` to `deleteContacts()`, should `setCertificateContacts()` and `getCertificateContacts()` also be renamed to `setContacts()` and `getContacts()`?

Reply:
cc @schaabs ^

Reply:
> deleteCertificateContacts to deleteContacts. Confirmed with Heath that this means that anything .*CertificateContacts.* needs to be .*Contacts.* except for the main class CertificateContacts

The above is from @sadasant if it helps

Reply:
This has been done with https://github.com/Azure/azure-sdk-for-js/pull/5991/commits/2fa9dfdb404e8e93af0b9288a80ee21a982d7162